### PR TITLE
Request is no longer blocking

### PR DESCRIPTION
**Pull request by Rackover**
Closes https://github.com/FAForever/server/issues/406

### CODE REVIEW

File: server/api/api_accessor.py [hunk 1]
```diff
@@ -98,11 +99,23 @@ def __init__(self):
 
     async def api_get(self, path):
         with self.api_session as api:
-            result = api.get(API_BASE_URL + path)
+            loop = asyncio.get_event_loop()
```
Comment (Askaholic) [inline server/api/api_accessor.py]: Could refactor to util function, something like:

```python
async def run_in_executor(fn):
    loop = asyncio.get_event_loop()
    future = loop.run_in_executor(None, fn)
    return await future
```

**Comment by Rackover** [comment]
Impossible, the token fetching is almost instantaneous, what is blocking here is the API achievements/events call

**Comment by Rackover** [comment]
Plus I'm not sure I can `with await self.api_session as api:`

**Comment by Askaholic** [comment]
You would have to use the async syntax: `async with self.api_session`

**Comment by Sheeo** [comment]
Might I suggest using [aiohttp](https://aiohttp.readthedocs.io/en/stable/) instead?

It's already in `requirements.txt`.

**Comment by Askaholic** [comment]
@Sheeo Sure, if someone has the motivation to write the oauth2 implementation.

**Comment by Rackover** [comment]
Superseded by https://github.com/FAForever/server/pull/433

**Pull request closed without merging**